Clinical-Genomics/scout
# Dismiss options for snv cancer variants

This PR adds 3 cancer-specific dismiss options to the select for dismissing variants on the cancer variant page.

Fixes the code so that dismissed snv cancer variants appear grey on the cancer variants page (fix 
#1614)
**How to test**:
1. Install a local implementation and load the demo cancer case with the following command: `scout --demo load case scout/demo/cancer.load_config.yaml`.

1. Go to cancer case and dismiss a variant with a specific cancer option (the 3 options at the end of the list in the dismiss select). 
1. Go back to the cancer variants page and make sure that the variant's line is grey

1. Check that dismissing a variant from the demo case (non-cancer) works and that the 3 cancer specific options are not present in the dismiss select of this case.

**Expected outcome**:
The functionality should be working as described.

**Review:**
